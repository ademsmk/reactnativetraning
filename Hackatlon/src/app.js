import React from 'react';
import {SafeAreaView, StyleSheet, View, ImageBackground} from 'react-native';

import {Login} from './pages/login';

export function App() {
  return (
    <ImageBackground
      source={require('../asset/run.jpg')}
      style={{width: '100%', height: '100%'}}>
      <SafeAreaView style={styles.safeArea}>
        <View style={styles.root}>
          <Login />
        </View>
      </SafeAreaView>
    </ImageBackground>
  );
}

const styles = StyleSheet.create({
  safeArea: {
    flex: 1,
  },
  root: {
    flex: 1,
  },
});
