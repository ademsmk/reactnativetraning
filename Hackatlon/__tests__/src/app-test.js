import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import { App } from "../../src/app";

describe('Login -> ./src/pages/login', () => {
    it(' should match snapshot', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
