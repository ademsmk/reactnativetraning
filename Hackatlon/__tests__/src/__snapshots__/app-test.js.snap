// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login -> ./src/pages/login  should match snapshot 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../asset/run.jpg",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignContent": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "center",
              "marginTop": 20,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "paddingLeft": 20,
              }
            }
          >
            <View
              accessible={true}
              focusable={false}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../asset/ok.png",
                  }
                }
                style={
                  Object {
                    "height": 30,
                    "width": 30,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignContent": "flex-end",
                "flex": 5,
                "flexDirection": "row",
                "justifyContent": "flex-end",
                "marginRight": 40,
              }
            }
          >
            <View
              accessible={true}
              focusable={false}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../asset/hearth.png",
                  }
                }
                style={
                  Object {
                    "height": 30,
                    "marginLeft": 20,
                    "width": 30,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={false}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../asset/upload.png",
                  }
                }
                style={
                  Object {
                    "height": 30,
                    "marginLeft": 20,
                    "width": 30,
                  }
                }
              />
            </View>
            <View
              accessible={true}
              focusable={false}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../asset/user.png",
                  }
                }
                style={
                  Object {
                    "height": 30,
                    "marginLeft": 20,
                    "width": 30,
                  }
                }
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 6,
            }
          }
        />
        <View
          style={
            Object {
              "flex": 5,
            }
          }
        >
          <View
            style={
              Object {
                "height": 30,
                "paddingBottom": 5,
                "paddingLeft": 20,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontFamily": "Arial",
                  "fontSize": 20,
                  "fontWeight": "600",
                }
              }
            >
              Intensity...
            </Text>
          </View>
          <View
            style={
              Object {
                "height": 30,
                "paddingBottom": 5,
                "paddingLeft": 20,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontFamily": "Arial",
                  "fontSize": 20,
                  "fontWeight": "600",
                }
              }
            >
              26 min | get In Shape
            </Text>
          </View>
          <View
            style={
              Object {
                "height": 60,
                "paddingBottom": 5,
                "paddingLeft": 20,
                "paddingTop": 10,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontFamily": "Arial",
                  "fontSize": 35,
                  "fontWeight": "600",
                }
              }
            >
              Full Body Flow
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "stretch",
                "flex": 1,
                "flexDirection": "row",
                "marginBottom": 50,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "margin": 20,
                  "marginTop": 50,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "opacity": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "alignItems": "center",
                      "backgroundColor": "white",
                      "borderWidth": 1,
                      "flexDirection": "row",
                      "height": 50,
                      "justifyContent": "center",
                    }
                  }
                >
                  <Image
                    source={
                      Object {
                        "testUri": "../../../asset/arrow.png",
                      }
                    }
                    style={
                      Object {
                        "height": 20,
                        "marginRight": 5,
                        "width": 20,
                      }
                    }
                  />
                  <Text
                    style={
                      Object {
                        "color": "black",
                        "fontFamily": "Arial",
                        "fontSize": 25,
                        "fontWeight": "400",
                        "textAlign": "center",
                        "textAlignVertical": "center",
                      }
                    }
                  >
                    Start
                  </Text>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTSafeAreaView>
</View>
`;
